docarray/helper.py

```diff
@@ -502,3 +502,15 @@ def login(interactive: Optional[bool] = None, force: bool = False, **kwargs):
 def logout():
     """Log out of Hubble account."""
     hubble.logout()
+
+
+def in_notebook() -> bool:
```

Comment:
There's an is_notebook method in hubble package. We should reuse it

Reply:
from hubble.utils.notebook import is_notebook

Reply:
> Do I just adjust this in the setup.py
yes I believe so